### DISCUSSION TITLE
Import GraphViz dot file

### DISCUSSION THREAD

**Discussion by Podbrushkin** [gephi/gephi#2790] [Q&A]
How to import data in GraphViz Dot format? Official webpage states it is supported, but in an hour or so I haven't found any slightest clue on how to import something like that:
```
digraph sample {
A -> B;
B -> C;
}
```
This page doesn't have any instructions: https://gephi.org/users/supported-graph-formats/graphviz-dot-format/
, .dot files are not recognized, I haven't found any official manual for the application, and there is no corresponding plugin in app's settings, and there is no videos on youtube about it.

**Comment by Podbrushkin** [comment]
Looks like Gephi actually can open the above dot example through File->Open..., if it's has a .dot extension. But for my .dot file it shows following error: 
```
FamilyTree - Copy.dot failed to import due to the following error: java.lang.RuntimeException: java.lang.IllegalArgumentException: The id can't be empty
```
even though this file is correctly formatted and can be shown in online graphviz tools. I assume it is a different topic.

**Reply by N0K0** [reply]
I got the same issue as i write this..
